.gitignore

```diff
@@ -6,3 +6,44 @@ nbproject/
 *.swp
 build/
 build_*/
+
+# Next.js project for webgui
```

Comment:
These new rules are specific to the `webgui-new` folder, and thus should be placed there.